modules/exploits/linux/http/trueonline_p660hn_v2_rce.rb

```diff
@@ -98,8 +98,8 @@ def exploit
     @cookie = rand_text_alpha_lower(7)
 
     res = send_request_cgi({
+      'method' => "GET",
```

Comment:
I'm trying to solve this info message :- modules/exploits/linux/http/trueonline_p660hn_v2_rce.rb - [INFO] Please use vars_get in send_request_cgi: send_request_cgi({ 'uri' => '/cgi-bin/index.asp?' + Rex::Text.encode_base64("#{datastore['USERNAME']